solidity/test/liquidation-tests/courtesyCallTest.js

```diff
@@ -0,0 +1,189 @@
+const { deployAndLinkAll } = require("./liquidation-test-utils/setup.js")
```

Comment:
@NicholasDotSol Do these tests have side-effects (eg. transfers of BTC persist between files/blocks), or do each get encapsulated in a snapshot? I know we recently completed a massive refactoring to the tests, so my understanding of what's going on might have changed here.

Reply:
there should be no side effects between tests

Reply:
> there should be no side effects between tests

Ah okay - yeah I ran them, can see that now. Do you know where this is implemented? ie. in openzeppelin